docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM bitnami/tomcat:7.0.94 as fineract
+
+USER root
+RUN apt-get update -qq && apt-get install -y git openjdk-8-jdk wget
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/jre
+
+RUN git clone https://github.com/apache/fineract.git
```

Comment:
hang on - why `git clone` here, instead of just what you have commented out, below? May be I am missing something about the intended "flow" of things here, but your doc says that as a user you have to "Clone the Fineract Github repository", so then you can just use the local sources directly (as you are in the commented out line 17), no?